### ISSUE TITLE
Windows Azure Guest agent: Supporting 32bit OS

### ISSUE THREAD

**Issue by albertogeniola** [MicrosoftDocs/SupportArticles-docs#210]
TLDR; Guest Agent support for 32bit is needed for a class of applications that cannot be run on 64bit systems.

Cloud adoption is often propelled by the need to empty some Datacenters and perform aggressive saving by dismissing expensive colocation contracts. That often require to comply with strict deadlines, which makes lift&shift the only viable option. 

When moving the entire DC in lift&shift, you may end up moving some legacy applications which require 32bit OSes. In particular, appliactions running on 16bit compatibility mode (old VB apps) will only run on 32bit OSes thanks to the NTVDM emulator, which is not available on any Windows 64bit system.

Azure correctly supports 32bit specialized images and that is really a relief as it enables lift&shift of 32bit legacy workloads. However, the Windows Azure Guest Agent does not support 32bit systems and that makes it hard to manage such cloud workloads: password reset, logging and monitoring.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c294ae1d-9518-2f87-0cc8-320b0cc5ac02
* Version Independent ID: 7be72203-e1f0-e692-399f-4a618d0b126e
* Content: [Install the Azure VM Agent in offline mode - Virtual Machines](https://docs.microsoft.com/en-us/troubleshoot/azure/virtual-machines/install-vm-agent-offline)
* Content Source: [support/azure/virtual-machines/install-vm-agent-offline.md](https://github.com/MicrosoftDocs/SupportArticles-docs/blob/main/support/azure/virtual-machines/install-vm-agent-offline.md)
* Service: **virtual-machines**
* GitHub Login: @genlin
* Microsoft Alias: **genli**

**Comment by genlin** [comment]
@albertogeniola  Currently Azure VM agent only supports 64-bit operating systems. I will forward your voice to our product team. Thanks.
